frontend/components/common.tsx

```diff
@@ -1,5 +1,4 @@
 import React from 'react'
-import PropTypes from 'prop-types'
 import styled, { css, createGlobalStyle } from 'styled-components'
 
```

Comment:
Question on the file extensions for my own edification: seems like a lot of these are going from `.js` to `.tsx`, but I suppose I was expecting `.jsx` --> `.tsx`. Are there some React-related changes we're adding here in addition to the TS switch? Or is `.tsx` extension just the way to go for TS files in React?

Reply:
> Or is `.tsx` extension just the way to go for TS files in React?

^^ Yes, TS files which include JSX need to be `.tsx`.

Once upon a time JS with JSX were `.jsx` but no longer. I'm not sure why TS has kept it this way.